### PR TITLE
Roll up complaints to all institutions with same ope

### PR DESCRIPTION
**Pull request by mphprogrammer**

### CODE REVIEW

File: spec/models/complaint_spec.rb [hunk 1]
```diff
@@ -99,22 +99,37 @@
...
-      weam = create :weam
-      crosswalk = create :va_crosswalk, facility_code: weam.facility_code, ope: "11111111"
-      create_list :complaint, 10, :all_issues, facility_code: weam.facility_code
+      weam0 = create :weam, facility_code: "00000000"
+      weam1 = create :weam, facility_code: "11111111"
+
+      crosswalk0 = create :va_crosswalk, facility_code: weam0.facility_code, ope: "22222222"
+      crosswalk1 = create :va_crosswalk, facility_code: weam1.facility_code, ope: "22222222"
+
+      # TODO Initialize by weans and update by crosswalk.
```
Comment (ayaleloehr) [inline spec/models/complaint_spec.rb]: This TODO looks like it's done with these two methods?  Can you explain more?

Comment (mphprogrammer) [inline spec/models/complaint_spec.rb]: Sorry, forgot to move the comment! It was done by the lines following ...

File: spec/models/complaint_spec.rb [hunk 2]
```diff
@@ -99,22 +99,37 @@
...
+      DataCsv.initialize_with_weams
+      DataCsv.update_with_crosswalk
+
+      # Generate 10 complaints for facility code "00000000", these should be reflected
+      # in ope6 sums for facility code "11111111".
+      create_list :complaint, 5, :all_issues, facility_code: weam0.facility_code
```
Comment (mphprogrammer) [inline spec/models/complaint_spec.rb]: Comment should read Generate 5 complaints

**Comment by ayaleloehr** [comment]
LGTM except for two comment changes that are needed. going to merge as they are not code changes and I would like to deploy tonight, but pinging @mphprogrammer to make those changes first thing tomorrow!

**Pull request merged**